README.md

```diff
@@ -1,24 +1,80 @@
-**STILL IN DEVELOPMENT, NOT FIT FOR GENERAL CONSUMPTION, DO NOT USE IF NOT A DEVELOPER!**
-
 # Slic3r plugin for OctoPrint
 
-**WARNING** Not functional yet since slic3r's return codes are behaving unexpectedly. WIP for others to take a look at
-and maybe build upon.
+This plugin has been successfully tested for Slicer's stable versions 1.1.7 and 1.2.9.
 
 ## Setup
 
-Install the plugin like you would install any regular Python package from source:
+In order to install the plugin, go to Settings -> Plugin Manager and click on 'Get more...':
+
+![Screenshot](http://imgur.com/9NaAl37.png)
+
+You'll see an option to add it from an URL. Add https://github.com/javierma/OctoPrint-Slic3r/archive/master.zip an click on 'Install' button.
+
+![Screenshot](http://i.imgur.com/lln2TvT.png)
+
+At this moment, the plugin will be installed, but Slic3r must be downloaded and configured, which can be done following these steps:
+
+  -If OctoPrint is running under a Raspberry Pi, you can use the following script to install Slic3r (unfortunately precompiled packages do not work for RPI's architecture). For that purpose, copy the following code in a file (i.e. slic3r_install.sh).
+  
+    #!/bin/sh
+
+    echo "Shell script created by Javier Martínez Arrieta for Slic3r installation\n"
+    #Ask the user for a version
+    echo "Please indicate which version you desire to be installed (e.g. 1.1.7)"
+    read version
+    echo "The installation of Slic3r takes a long time. PLease be patient"
+    cd $HOME
+    echo "Installing required libraries and dependencies..."
+    sudo apt-get install git libboost-system-dev libboost-thread-dev git-core build-essential libgtk2.0-dev libwxgtk2.8-dev libwx-perl libmodule-build-perl libnet-dbus-perl cpanminus libextutils-cbuilder-perl gcc-4.7 g++-4.7 libwx-perl libperl-dev
+    sudo cpanm AAR/Boost-Geometry-Utils-0.06.tar.gz Math::Clipper Math::ConvexHull Math::ConvexHull::MonotoneChain Math::Geometry::Voronoi Math::PlanePath Moo IO::Scalar Class::XSAccessor Growl::GNTP XML::SAX::ExpatXS PAR::Packer
+    echo "Cloning Slic3r repository..."
+    git clone https://github.com/alexrj/Slic3r.git
+    cd Slic3r
+    git checkout $version
+    echo "Building and testing Scli3r..."
+    sudo perl Build.PL
+    echo "If everything was installed properly,you should be able to run Slic3r with the command ./slic3r.pl"   
+  
+  -Save the file and allow execution giving permission. Using a terminal, it would be chmod +x slic3r_install.sh (or the name given to the file).
+  
+  ![Screenshot](http://imgur.com/NRZkshW.png)
```

Comment:
Command line input (and output) in a setup guide should not be made through a screenshot. You have code environments in markdown for stuff like that, e.g. this

    ```
    chmod +x slic3r_install.sh
    ```

will turn into

>     chmod +x slic3r_install.sh

when rendered. Same applies for the following steps.

Your use of `-` here also produces weird looking output. Use proper markdown enumerations:

    1. foo
    2. bar
    3. fnord

=>

> 1. foo
> 2. bar
> 3. fnord

Makes the documentation *way* more readable :)